Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 # 5.0.0
 
+- Updated to latest ReactiveCocoa v4.0-alpha.4
```

Comment:
Hey! Sorry for the confusion, but this should go in the "Next" section, since 5.0.0 is already released. Does that make sense?